packages/run-it/src/components/common/common.tsx

```diff
@@ -24,7 +24,7 @@
 
  */
 import styled from 'styled-components'
-import { Heading } from '@looker/components'
+import { Heading, Span } from '@looker/components'
```

Comment:
I'm looking at the components in this file and i'm not seeing a lot of meat. Not totally convinced but I can sort of see it. I vaguely remember that defaultProps caused problems for tree shaking so this will probably need to be redone. 

I see why its in run it but maybe its time to create an "sdk-components" package (for want of a better name).

Not saying part if this PR. Just a general observation.

Reply:
Yeah, definitely worth considering.

Reply:
This becomes even more valuable when we lazy load the various response renderers